test/views/inp.njk

```diff
@@ -89,7 +89,7 @@
       inp.entries = inp.entries.map((e) => ({
         ...e.toJSON(),
         interactionId: e.interactionId,
-        target: e.target.nodeName.toLowerCase(),
+        target: e.target?.nodeName.toLowerCase(),
```

Comment:
Is this related to FID entry?

Reply:
Yeah, not sure why but I started noticing lots of cases where `target` was undefined on `first-input` entries. I didn't look much further into repo-ing why.